docs/source/tutorials/two_class_microscopy_seg_2d_unet.rst

```diff
@@ -0,0 +1,305 @@
+First off, copy this configuration file in your local directory (to avoid modifying the source file):
+
+.. code-block:: bash
+
+   cp <PATH_TO_IVADOMED>/ivadomed/config/config_microscopy.json .
+
+Then, open it with a text editor. Below we will discuss some of the key parameters to perform a two-class 2D
+microscopy segmentation training.
+
+- ``command``: Action to perform. Here, we want to train a model, so we set the fields as follows:
+
+  .. code-block:: xml
+
+     "command": "train"
+
+- ``path_output``: Folder name that will contain the output files (e.g., trained model, predictions, results).
+
+  .. code-block:: xml
+
+     "path_output": "log_microscopy_sem"
+
+- ``loader_parameters:path_data``: Location of the dataset. As discussed in `Data <../data.html>`__, the dataset
+  should conform to the BIDS standard. Modify the path so it points to the location of the downloaded dataset.
+
+  .. code-block:: xml
+
+     "path_data": "data_axondeepseg_sem"
+
+- ``loader_parameters:bids_config``: Location of the custom BIDS configuration file required for microscopy
+  file indexing.
+
+  .. code-block:: xml
+
+     "bids_config": "ivadomed/config/config_bids.json",
+
+- ``loader_parameters:target_suffix``: Suffix of the ground truth segmentations. The ground truths are located
+  under the ``data_axondeepseg_sem/derivatives/labels`` folder. In our case, the suffix are ``_seg-axon-manual``
+  and ``_seg-myelin-manual``:
+
+  .. code-block:: xml
+
+     "target_suffix": ["_seg-axon-manual", "_seg-myelin-manual"]
+
+- ``loader_parameters:extensions``: File extensions of the microscopy raw data
+
+  .. code-block:: xml
+
+     "extensions": [".png"]
+
+- ``loader_parameters:contrast_params``: Contrasts of interest. In our case, we are training a single contrast model
+  with contrast ``SEM``.
+
+  .. code-block:: xml
+
+     "contrast_params": {
+         "training_validation": ["SEM"],
+         "testing": ["SEM"],
+         "balance": {}
+     }
+
+- ``loader_parameters:slice_axis``: Orientation of the 2D slice to use with the model.
+  2D PNG files must use default ``axial``.
+
+  .. code-block:: xml
+
+     "slice_axis": "axial"
+
+- ``split_dataset:split_method``: Describe the metadata used to split the train/validation/test sets.
+  Here, ``sample_id`` from the ``samples.tsv`` file will shuffle all samples, then split them between
+  train/validation/test sets.
+- ``split_dataset:train_fraction``: Fraction of the dataset's ``sample_id`` in the train set. In our case ``0.6``.
+- ``split_dataset:test_fraction``: Fraction of the dataset's ``sample_id`` in the test set. In our case ``0.1``.
+
+  .. code-block:: xml
+
+      "split_method" : "sample_id"
+      "train_fraction": 0.6
+      "test_fraction": 0.1
+
+- ``training_time:num_epochs``: The maximum number of epochs that will be run during training. Each epoch is composed
+  of a training part and an evaluation part. It should be a strictly positive integer. In our case, we will use
```

Comment:
This is just a suggestion as to me evaluation sounds more on the test set and validation on the validation set, but please ignore if you disagree. Later in the tutorial, there is a section named `Evaluate model` which refers to inference on the testing set. Using evaluation here might be confusing.

```suggestion
  of a training part and a validation part. It should be a strictly positive integer. In our case, we will use
```

Reply:
Agreed!